app.js

```diff
@@ -43,17 +59,33 @@ let bullet = {
   rotation: 0,
 };
 
-let enemy2X = 1600;
-let enemy3X = 2600;
-let enemy2Y = 81;
-let enemy3Y = 281;
-let rocketX = 20;
-let rocketY = 70;
+let isShipMovingUp = false;
+let isShipMovingLeft = false;
+let isShipMovingRight = false;
+let isShipMovingDown = false;
+let enemyHealth50 = {
+  element: document.querySelector('#healthBar50')};
+let enemyHealth100 = {
+  element: document.querySelector('#healthBar100')};
+let enemyHealth200 = {
+  element: document.querySelector('#healthBar200')};
+let enemyHealth50text = {
+  element: document.querySelector('#healthBar50text')};
+let enemyHealth100text = {
+  element: document.querySelector('#healthBar100text')};
+let enemyHealth200text = {
+  element: document.querySelector('#healthBar200text')};
 let rocketVelocity = 0;
+let gutlingVelocity = -2;
 let enemy1Velocity = -2;
 let enemy2Velocity = -2;
 let enemy3Velocity = -2;
-let rocket = document.querySelector("#rocket");
+let enemyHealth50Velocity = -2;
```

Comment:
I suggest to put `velocity` inside the `enemyHealth50` object.
Like this:
```
let enemyHealth50 = {
  element: document.querySelector('#healthBar50'),
  velocity: -2,
};
```
So we will have 1 variable instead of 2.